README.md

```diff
@@ -2,5 +2,6 @@
 
 Client-side JavaScript library that can be used to secure web applications and integrate them with Keycloak.
 
-TODO
+The documentation can be found in the [Keycloak documentation](https://www.keycloak.org/securing-apps/javascript-adapter).
 
+TODO
```

Comment:
Can we remove this TODO, or should we be adding more?

Reply:
I can remove it and add the proper link to the current documentation: https://www.keycloak.org/securing-apps/javascript-adapter

Reply:
Sounds good to me :)

Reply:
I have followed your advise and the README is now similar to the used in keycloak-nodejs-connect repo.